src/aggregate_loader.py

```diff
@@ -1,178 +1,217 @@
-        }
-
-        # This is the order in which data types are loaded.
-        # After each list, the loader will "pause" and wait for that item to finish.
-        # i.e. After Ontology, there will be a pause.
-        # After GO, DO, MI, there will be a pause, etc.
-        list_of_etl_groups = [
-            ['DOID', 'MI'],
-            ['GO'],
-            ['ONTOLOGY'],
-            ['ECOMAP'],
-            ['BGI'],
-            ['CONSTRUCT'],
-            ['ALLELE'],
-            ['VARIATION'],
-            ['SQTR'],
-            ['AGM'],
-            ['GFF'],
-            ['EXPRESSION'],
-            ['ExpressionRibbon'],
-            ['ExpressionRibbonOther'],
-            ['GENEEEXPRESSIONATLASSITEMAP'],
-            ['DAF'],  # Locks Genes
-            ['PHENOTYPE'],  # Locks Genes
-            ['ORTHO'],  # Locks Genes
-            ['GAF'],  # Locks Genes
-            ['GeoXref'],  # Locks Genes
-            ['GeneDiseaseOrtho'],
-            ['INTERACTION-MOL'],
-            ['Closure'],
-            ['GeneDescriptions'],
-            ['VEP'],
-            ['VEPTRANSCRIPT']
-        ]
         etl_time_tracker_list = []
-
-        for etl_group in list_of_etl_groups:
+        for etl_group in cls.etl_groups:
             etl_group_start_time = time.time()
             logger.info("Starting ETL group: %s" % etl_group)
             thread_pool = []
             for etl_name in etl_group:
                 logger.info("ETL Name: %s" % etl_name)
                 config = data_manager.get_config(etl_name)
                 if config is not None:
-                    etl = etl_dispatch[etl_name](config)
-                    p = multiprocessing.Process(target=etl.run_etl)
-                    p.start()
-                    thread_pool.append(p)
+                    etl = cls.etl_dispatch[etl_name](config)
+                    process = multiprocessing.Process(target=etl.run_etl)
+                    process.start()
+                    thread_pool.append(process)
                 else:
                     logger.info("No Config found for: %s" % etl_name)
             ETL.wait_for_threads(thread_pool)
-                
+
             logger.info("Waiting for Queues to sync up")
-            nt.check_for_thread_errors()
-            nt.wait_for_queues()
+            neo_transactor.check_for_thread_errors()
+            neo_transactor.wait_for_queues()
             etl_elapsed_time = time.time() - etl_group_start_time
-            etl_time_message = ("Finished ETL group: %s, Elapsed time: %s" % (etl_group, time.strftime("%H:%M:%S", time.gmtime(etl_elapsed_time))))
+            etl_time_message = ("Finished ETL group: %s, Elapsed time: %s"
+                                % (etl_group,
+                                   time.strftime("%H:%M:%S", time.gmtime(etl_elapsed_time))))
             logger.info(etl_time_message)
             etl_time_tracker_list.append(etl_time_message)
 
-        nt.shutdown()
```

Comment:
why is this removed? just curious for more detail

Reply:
Good question. It is not removed it is just now on line 205